src/index.js

```diff
@@ -1,20 +1,22 @@
-const js_hamburger = document.getElementById("js-hamburger");
-const js_globalNav = document.getElementById("js-global-nav");
+const hamburger = document.getElementById("js-hamburger");
+const globalNav = document.getElementById("js-global-nav");
 
-js_hamburger.addEventListener("mouseover",function() {
-  js_globalNav.style.willChange = "transform";
+
+hamburger.addEventListener("mouseover",function() {
+  globalNav.style.willChange = "transform";
 });
-js_hamburger.addEventListener("mouseout",function() {
-  js_globalNav.style.willChange = "";
+hamburger.addEventListener("mouseout",function() {
+  globalNav.style.willChange = "";
 });
 
-js_hamburger.addEventListener("click",function () {
-  js_globalNav.classList.toggle("is-active");
-  if (this.getAttribute("aria-expanded") === "false") {
+hamburger.addEventListener("click",function () {
+  globalNav.classList.toggle("is-active");
+  if (globalNav.classList.contains("is-active")) {
+
```

Comment:
IMO
this [code](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/js/accordion.js) is `wai-aria-practices-1.1/examples/accordion`

using `contain` to find class name.

```js
  accordion.addEventListener('click', function (event) {
    var target = event.target;

    if (target.classList.contains('Accordion-trigger')) {
      // Check if the current toggle is expanded.
```

Reply:
By the way, Are you going to apply araira to all future code?

Reply:
Thank you very much.
I'm learning so much.

Reply:
> By the way, Are you going to apply aria and FLOCSS to all future code?

I will continue to use FLOCSS.
I will continue to use FLOCSS because I am researching it.
I also use FLOCSS in my work.

I don't intend to use the aria attribute for everything.
This time, I used it even though I didn't understand it very well...

Reply:
> I don't intend to use the aria attribute for everything.
This time, I used it even though I didn't understand it very well...

ok. i understand.

> I will continue to use FLOCSS.
I will continue to use FLOCSS because I am researching it.
I also use FLOCSS in my work.

ok.
iBEM is recommended for the more popular types,
Personally, BEM is not needed to learn the front-end.